kbcstorage/workspaces.py

```diff
@@ -0,0 +1,112 @@
+"""
+Manages workspace requests to the API.
+
+Full documentation `here`.
+
+.. _here:
+    http://docs.keboola.apiary.io/#reference/workspaces/
+"""
+
+from kbcstorage.base import Endpoint
+
+
+def _make_body(mapping):
+    """
+    Given a dict mapping Keboola tables to aliases, construct the body of
+    the HTTP request to load said tables.
+
+    Args:
+        mapping(:obj:`dict`): Keys contain the full names of the tables to
+            be loaded (ie. 'in.c-bucker.table_name') and values contain the
+            aliases to which they will be loaded (ie. 'table_name').
```

Comment:
I definitely want to support those additional parameters because it's a use case I require personally but I don't think this is fully documented. I have [link](https://p4.zdusercontent.com/attachment/375538/islx580CMndAwT2mYEWWmGt4p?token=eyJhbGciOiJkaXIiLCJlbmMiOiJBMTI4Q0JDLUhTMjU2In0..Dg50uUBepG2etWCLhNIqmg.iaanMh-UEZDnBtgcuYXzgMeYkiT5e_6jOIU7OlkLzCC31pez29rzKUX-mcd5lF4SEJZ7hdzhUlrOQS53MOfhU0JRlBHLOK9h4V5smgQQzqIV9d3CDAp_jzro457H1ShI-fqmG2uGW76YoYeOJi3cXfA-k0CMr31C3goXbPRD2dyt9eJlu45Ri40RriTxpR_7OQzbZJDVa2FFK_pPsVr5X5N5drxUr4ujvPTfdhEs9kE5AFl5HolLB4jMl8ujqgHSSRp3yUZ94yNGTHif1baPkw.1sXDpvVaBGMXx_6jfKd_bA) from keboola support which is helpful of course, but would be good to have a full spec.